spec/pushmi_pullyu/swift_depositer_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+RSpec.describe PushmiPullyu::SwiftDepositer do
+  it 'not valid with nil arguments' do
+    expect { described_class.new(nil, nil) }.to raise_error(RuntimeError)
+  end
+
+  it 'valid connection established' do
+    VCR.use_cassette('swift_connect') do
+      swift_connection = described_class.new({ username: 'test:tester',
+                                               password: 'testing',
+                                               tenant: 'tester',
+                                               endpoint: 'http://www.example.com:8080',
+                                               auth_version: 'v1.0' },
+                                             'ERA')
+      expect(swift_connection).not_to be_nil
+      expect(swift_connection).to be_an_instance_of(described_class)
+      expect(swift_connection.swift_connection).to be_an_instance_of(OpenStack::Swift::Connection)
+    end
+  end
+
+  it 'deposit file' do
+    VCR.use_cassette('swift_deposit') do
+      swift_connection = described_class.new({ username: 'test:tester',
+                                               password: 'testing',
+                                               tenant: 'tester',
+                                               endpoint: 'http://www.example.com:8080',
+                                               auth_version: 'v1.0' },
+                                             'ERA')
+
+      sample_file = "#{File.dirname(__FILE__)}/../fixtures/config.yml"
```

Comment:
you can just do `sample_file = 'spec/fixtures/config.yml' `. Much nicer then having in my opinion.